### PR TITLE
Link to official English language rules

### PR DESCRIPTION
**Pull request by ChrisTheCat**

### CODE REVIEW

File: lib/engine/game/g_18_mag/meta.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ module Meta
         GAME_INFO_URL = 'https://github.com/tobymao/18xx/wiki/18Mag'
         GAME_LOCATION = 'Hungary'
         GAME_PUBLISHER = :lonny_games
-        GAME_RULES_URL = 'https://www.lonny.at/games/18magyarorsz%C3%A1g/'
+        GAME_RULES_URL = 'https://boardgamegeek.com/filepage/222188/18mag-rules-english'
```
Comment (roseundy) [inline lib/engine/game/g_18_mag/meta.rb:18]: Is there a reason not to use the official rules from Lonny?
https://www.lonny.at/app/download/10197748784/18Mag_Rules_ENG_comp.pdf?t=1620544783

**Comment by ChrisTheCat** [comment]
That would probably be fine too. I didn't know that file existed. Lonny
just posted the final rules to BGG. Previously, the link went to a page on
his website that didn't actually have the final rules.

**Comment by roseundy** [comment]
Let's use the official one from Lonny, since not everyone has BGG file access.  This link works too:
https://www.lonny.at/app/download/10197748784/18Mag_Rules_ENG_comp.pdf

**Comment by tobymao** [comment]
can someone update this and merg?

**Pull request merged**